docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+
+services:
+    db:
+        image: "mariadb:bionic"
+        container_name: "rathena_db"
+        ports:
+            - "3306:3306" # allow DB connections from host
+        volumes:
+            - "rathenadb:/var/lib/mysql" # save database to local disk
+            - "./sql-files/:/docker-entrypoint-initdb.d" # initialize db with ./sql-files
+        environment:
+            MYSQL_ROOT_PASSWORD: ragnarok
+            MYSQL_DATABASE: ragnarok
+            MYSQL_USER: ragnarok
+            MYSQL_PASSWORD: ragnarok
+    server:
```

Comment:
I think I would have split server across 3 images to be a bit more flexible

Reply:
That works too, I see the pros here being STDOUT logs are more distinctly separated and being able to specify different resource limits per server. Cons that I can think of is perhaps slightly more overhead of having more containers running.